content/security-humanops.rst

```diff
@@ -0,0 +1,231 @@
+
+If possible, use non-password options.
+
+Use password management tools (ie LastPass, 1Password, etc) and ensure to enable MFA for these tools.
+
+Do not reuse passwords from your personal accounts with SFTPPlus accounts.
+
+Do not send passwords via cleartext. Or even better, do not continue to use services that send passwords via cleartext.
+
+When changing passwords do not use common password mutations like changing a character at the end.
+
+Do not allow opportunities for password profiling, like using company products in your password.
+
+
+Internet facing information
+===========================
+
+Consider what services are facing the Internet.
+
+Consider what information is facing the Internet.
+
+When posting reviews or comments, are there certain details that need to be taken out to sanitize the review or comment?  
+
+Is this public information going to help with someone's enumeration activities? Are there sensitive files, passwords, private key info publicly facing? 
+
+
+Customers
+=========
+
+We work with admins that are already aware of security practices - like not leaving sensitive information in logs.
+
+However we work with a number of customer bases and budgets.
+
+
+Olark
+-----
+
+We use Olark web chat. If talking to a customer, please verify who they are and their installation.  The other alternative is to move the discussion to email.
+
+
+Improving Documentation
+-----------------------
+
+If there is a customer having trouble securing their setup and they have read the documentation, ask what they are trying to do.  Then detect what portion of the documentation can be improved and create a ticket for this.
+
+If a customer has gone to support for this issue and it is not an 'edge case', then chances are high that others have had this issue before.
+
+
+Testing / validating third party apps / software for use
+========================================================
+
+There are some instances when we need to set up test accounts - for example to trial a ticketing software, an online web software and so on. When doing a test, try to test using your own individual Pro:Atria email as it helps solve 'noise'.
+
+Take note of what third party services are opened to do testing in the internal wiki.  
+
+If the third party service is no longer being used, or when testing is finished, delete all instances of any irrelevant tests/accounts and unused services.  Then update the wiki with your findings.
+
+If an app is in active use with customers / within the team during the validating period, ensure that whitelisting takes place to help validate/verify that it is an Pro:Atria-used third party product.
+
+
+Traveling, use of multiple devices etc
+=======================================
+
+If you are planning to travel while working, there are additional precautions faced being on the move.  Anything from risk of using hostile networks, to risk of device being stolen/tampered with.  
```

Comment:
I don't think that this is useful, even if you are not traveling you should still protect your connection

and the device can also be stolen from home... co work... etc